sunpy/visualization/animator/tests/test_basefuncanimator.py

```diff
@@ -180,3 +181,23 @@ def test_lineanimator_figure():
     fig = plt.figure()
     ani = LineAnimator(data0, plot_axis_index=plot_axis0, axis_ranges=[None, xdata], fig=fig)
     ani.plot_start_image(ax=fig.gca())
+
+
+@figure_test
+def test_imageanimator_figure():
+    from itertools import product
```

Comment:
can you move the imports to the top of the file?

Reply:
Yup, I will.